app/models/user.rb

```diff
@@ -2,6 +2,15 @@ class User < ApplicationRecord
   attr_accessor :remember_token, :activation_token, :reset_token
 
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
+                                  foreign_key: "follower_id",
+                                  dependent: :destroy
```

Comment:
```suggestion
  has_many :active_relationships, class_name: Relationship.name,
                                  foreign_key: :follower_id,
                                  dependent: :destroy
```
class_name, foreign_key để làm gì e ?

Reply:
ví dụ ta có trong class User ta có association: has_many: posts => Rail sẽ ngầm hiểu class_name: Post và FK: user_id  
nhưng ở đây t đặt association name là active_relationships => ta phải chỉ rõ là class_name và fk nào (do ko thể suy ra từ active_relationships)